include/metal/list/cascade.hpp

```diff
@@ -59,14 +59,12 @@ namespace metal {
 namespace metal {
     /// \cond
     namespace detail {
-        template<typename outer, typename inner>
+        template<class outer, class inner>
         struct _cascader {};
 
-        template<
-            template<typename...> class outer,
-            template<typename...> class inner>
+        template<template<class...> class outer, template<class...> class inner>
```

Comment:
nice

Reply:
Here, I was not sure which one I prefer: the old version had a nice alignment that highlighted the symmetry of `outer` vs `inner`.

Anyway, I surely do not want to introduce manual formatting for these details, and I definitely prefer `class` over `typename`.